python-large-file-upload/notecardDataTransfer.py

```diff
@@ -4,26 +4,28 @@
 
 
 DEFAULT_WEB_TRANSACTION_TIMEOUT_SEC = 30
+DEFAULT_CARD_WEB_REQUEST = "web.post"
 
 
 class BinaryDataUploader:
 
-    webReqRoot = {"req":"web.post",
+    webReqRoot = {"req":DEFAULT_CARD_WEB_REQUEST,
                 "seconds": DEFAULT_WEB_TRANSACTION_TIMEOUT_SEC,
                 "content":"application/octet-stream",
                 "binary":True,
                 "route": '',
                 "offset":0,
                 "total":0
                 }
-    
+
     ConnectionTimeoutSeconds = 90
 
-    def __init__(self, card, route, timeout=DEFAULT_WEB_TRANSACTION_TIMEOUT_SEC, printFcn=print, waitForNotehubConnection=True, setTempContinuousMode=True) -> None:
+    def __init__(self, card, req, route, timeout=DEFAULT_WEB_TRANSACTION_TIMEOUT_SEC, printFcn=print, waitForNotehubConnection=True, setTempContinuousMode=True) -> None:

```

Comment:
Not a requirement by any means, but you could make `req` an optional argument.

```
def __init__(self, card, route, req = DEFAULT_CARD_WEB_REQUEST, timeout=DEFAULT_WEB_TRANSACTION_TIMEOUT_SEC, ...
```

That way it will default to using `web.post`  if no req argument is supplied.

Certainly not a big deal, just wanted to point it out as an option as an opportunity to learn more about Python.

If you want the `req` to always be required by design, then you already have the correct function signature!